addons/colorblind/addon.json

```diff
@@ -0,0 +1,92 @@
+{
+  "name": "Color blindness accommodations",
+  "description": "A few features to help make Scratch more accessible to colorblind users.",
+  "credits": [
+    {
+      "name": "DNin01",
+      "link": "https://github.com/DNin01"
+    }
+  ],
+  "customCssVariables": [
+    {
+      "name": "tdecoOverride",
+      "value": {
+        "type": "map",
+        "source": {
+          "type": "settingValue",
+          "settingId": "links"
+        },
+        "options": {
+          "underline": "underline"
+        }
+      }
+    }
+  ],
+  "userstyles": [
+    {
+      "url": "bold-links.css",
+      "matches": ["https://scratch.mit.edu/*"],
+      "if": {
+        "settings": { "links": "bold" }
+      }
+    },
+    {
+      "url": "underline-links.css",
+      "matches": ["https://scratch.mit.edu/*"],
+      "if": {
+        "settings": { "links": "underline" }
+      }
+    }
+  ],
+  "settings": [
+    {
+      "name": "Link style",
+      "id": "links",
+      "type": "select",
+      "potentialValues": [
+        {
+          "id": "none",
+          "name": "Default"
+        },
+        {
+          "id": "bold",
+          "name": "Bold"
+        },
+        {
+          "id": "underline",
+          "name": "Underlined"
+        }
+      ],
+      "default": "bold"
```

Comment:
I think "bold" is the option that makes the least sense.

We can choose "default" and make this behave like other addons named "customizable" (no changes by default), or have "underline" as a default which lets you easily notice what the addon changed after enabling it